src/operation.cpp

```diff
@@ -11,24 +11,26 @@
 #include <tf2/LinearMath/Quaternion.h>
 #include <tf2/transform_datatypes.h>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf/transform_broadcaster.h>
```

Comment:
Unnecessary include? I couldn't find any other changes indicating the necessity of this include.

Reply:
Didn't see all the changes when I reviewed, so I might very well be wrong.

Reply:
Very good point. I tried some stuff and forgot to remove it.
But that also highlighted quite many other useless includes there!!